c2corg_api/views/user.py

```diff
@@ -113,6 +122,24 @@ def validate_forum_username(request, **kwargs):
             request.errors.add('body', attrname, res)
 
 
+def validate_username(request, **kwargs):
+    """
+    Check that the username is not an email,
+    or that it is the same as the actual email.
+    """
+    attrname = 'username'
+    if attrname in request.json and 'email' in request.json:
+        value = request.json[attrname]
+        email = request.json['email']
+        if (valid_email(value) and email != value):
+            request.errors.add(
+                'body',
+                attrname,
+                'You cannot use someone else\'s email as username')
```

Comment:
That could be your email address though, for instance if you have several...
The message should rather be something like:
`An email address used as username should be the same as the one used as the account email address.`.

Reply:
k